src/DataSource.ts

```diff
@@ -12,14 +11,14 @@ import { DataSourceWithBackend, getTemplateSrv } from '@grafana/runtime';
 import { GithubDataSourceOptions, Label, GitHubQuery, GitHubVariableQuery } from './types';
 import { replaceVariables } from './variables';
 import { isValid } from './validation';
-import { getAnnotationsFromFrame } from 'common/annotationsFromDataFrame';
 
 export class DataSource extends DataSourceWithBackend<GitHubQuery, GithubDataSourceOptions> {
   constructor(instanceSettings: DataSourceInstanceSettings<GithubDataSourceOptions>) {
     super(instanceSettings);
   }
 
   templateSrv = getTemplateSrv();
+  annotations = {};
```

Comment:
to migrate old queries... you can implement:
https://github.com/grafana/grafana/blob/master/packages/grafana-data/src/types/annotations.ts#L73